SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -222,16 +266,15 @@ def customise_Reco(process,pileup):
 
     process.reconstruction.remove(process.castorreco)
     process.reconstruction.remove(process.CastorTowerReco)
-    process.reconstruction.remove(process.ak7CastorJets)
-    process.reconstruction.remove(process.ak7CastorJetID)
+    #process.reconstruction.remove(process.ak7BasicJets)
+    #process.reconstruction.remove(process.ak7CastorJetID)
 
     #the quadruplet merger configuration     
-    # from this PSet the quadruplet merger uses only the layer list so these could probably be removed
-    from RecoPixelVertexing.PixelTriplets.quadrupletseedmerging_cff import PixelSeedMergerQuadruplets
-    PixelSeedMergerQuadruplets.BPix.TTRHBuilder = cms.string("PixelTTRHBuilderWithoutAngle" )
-    PixelSeedMergerQuadruplets.BPix.HitProducer = cms.string("siPixelRecHits" )
-    PixelSeedMergerQuadruplets.FPix.TTRHBuilder = cms.string("PixelTTRHBuilderWithoutAngle" )
-    PixelSeedMergerQuadruplets.FPix.HitProducer = cms.string("siPixelRecHits" )
+    process.load("RecoPixelVertexing.PixelTriplets.quadrupletseedmerging_cff")
+    process.PixelSeedMergerQuadruplets.BPix.TTRHBuilder = cms.string("PixelTTRHBuilderWithoutAngle" )
+    process.PixelSeedMergerQuadruplets.BPix.HitProducer = cms.string("siPixelRecHits" )
+    process.PixelSeedMergerQuadruplets.FPix.TTRHBuilder = cms.string("PixelTTRHBuilderWithoutAngle" )
+    process.PixelSeedMergerQuadruplets.FPix.HitProducer = cms.string("siPixelRecHits" )    

```

Comment:
@boudoul I think this piece should be left as it was (but see below). The configuration of seeding layers was changed in 71X (#2378), and although at the time I didn't test any upgrade workflows, I'm quite confident that the existing configuration would work (or I would like to see the error message before "fixing" them).

The comment ("# from this PSet...") could actually be removed (being misleading), because the modifications **do** affect these parameters for e.g. `initialStepSeeds` (something what I didn't realize back then).

On the other hand, it could be more clear (or consistent) to add the `PixelSeedMergerQuadruplets` PSet to `process`, and then access it via the `refToPSet_` mechanism (as we already do for e.g. trajectory filter and builder configurations). Then the diff above would be fine, and we should change all `layerList = PixelSeedMergerQuadruplets` to `layerList = cms.PSet(refToPSet_ = cms.string("PixelSeedMergerQuadruplets"))`.